### ISSUE TITLE
No matching route found for custom theme css

### ISSUE THREAD

**Issue by siiron** [frctl/fractal#214]
I'm having trouble using custom themes. I'm trying to customize a theme following the guidelines found at [http://fractal.build/guide/customisation/web-themes#template-customisation](http://fractal.build/guide/customisation/web-themes#template-customisation).

My current structure looks like this:

```
├── src/
    ├── assets/
    ├── components/
    ├── docs/
├── theme/
    ├── assets/
        ├── favicon.ico (override)
    ├── css/
        ├── mytheme.css (override)
    ├── views/
        ├── partials/
            ├── header.nunj (override)
fractal.js
gulpfile.js
```

My theme configuration in `fractal.js` is as follows:
```js

const mandelbrot = require('@frctl/mandelbrot');

const myTheme = mandelbrot({
    favicon: '/assets/favicon.ico',
    skin: 'white',
    nav: ['docs', 'components'],
    styles: [
        'default',
        '/css/mytheme.css'
    ]
});

myTheme.addLoadPath(__dirname + '/theme');
fractal.web.theme(myTheme);

```

When running `serve` and opens http://localhost:3000, I get the following errors in the terminal:
```
✔ Fractal server is now running at http://localhost:3000
[11:16:18] Finished 'fractal:serve' after 2.14 s
✘ No matching route found for /css/mytheme.css
✘ Asset source not found
```

So basically, I get a 404 on these files. In addition, my custom header.nunj is not being used instead of the default one from mandlebrot.

Should the `theme/`-folder live inside `src` in this case, or does it not matter as I define the path in `addLoadPath`.

I've also studied the 24ways version of Fractal, but they just reference a css-file within the `src/assets/styles/`-folder [see fractal.js](https://github.com/24ways/frontend/blob/master/fractal.js#L12). This is not really what I want. I'd rather separate overrides of the Web UI to it's separate folder.

Any examples of github-repos with both custom css and overriding views would be much obliged.

For the record, I'm running:
Node v.6.9.5
Fractal v.1.1.3
Mandelbrot v.1.1.0

**Comment by allmarkedup** [comment]
Hey @siiron, sorry for the slow reply. It sounds like your assets are not being served up - even when customising a theme you still need to tell Fractal about where your static assets live: http://fractal.build/guide/web#static-assets

If your theme assets are in a different location from the component library assets, you can add another static asset path via the `theme..addStatic()` method. In your case, I think this would look like: 

```js
myTheme.addStatic(__dirname + '/theme/css/mytheme.css'); 
```

Hopefully that helps a little!

**Comment by siiron** [comment]
Thanks!

**Comment by ammara121** [comment]
https://github.com/ammara121/programmer/issues/1

Here is my probelm....
plz help

**Comment by allmarkedup** [comment]
An update on this issue for anyone who stumbles across it - the answer I posted above is totally incorrect! The `addStatic` call should point to the _directory_ of static files that need to be served, not to the individual file itself.

So the correct path for the example above is actually:

```js
myTheme.addStatic(__dirname + '/theme');
```

**Comment by Wursthandschuh** [comment]
`const mytheme= mandelbrot({
    styles: [
        "default",
        "fractal.css"
    ]
});

vrnTheme.addStatic(__dirname + '/themes/mytheme/dist/css'); `

this works on overview page but not on /components/mycomponent

anyone know why?

**Comment by LeBenLeBen** [comment]
@Wursthandschuh Can you try adding a slash before the filename `fractal.css` ?

**Comment by Wursthandschuh** [comment]
> @Wursthandschuh Can you try adding a slash before the filename `fractal.css` ?

this resolved the issue.
Thx for the instant reply
